acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -639,11 +905,14 @@ class N5GenerationParameters(argschema.schemas.DefaultSchema):
     chunk_size = argschema.fields.Tuple((
         argschema.fields.Int(),
         argschema.fields.Int(),
-        argschema.fields.Int()), required=False, default=(64, 64, 64))
+        argschema.fields.Int(),
+        argschema.fields.Int(),
+        argschema.fields.Int()), required=False, default=(1, 1, 64, 64, 64))
     mip_dsfactor = argschema.fields.Tuple((
         argschema.fields.Int(),
         argschema.fields.Int(),
         argschema.fields.Int()), required=False, default=(2, 2, 2))
+    deskew = argschema.fields.Str(required=False, default='')
```

Comment:
Not sure this makes sense in `DownsampleOptions`.  Should certainly have its own schema, but we should also figure out where to nest or subclass from.

Reply:
Yes, I'll set up a separate schema for the deskew.